src/compiler.mjs

```diff
@@ -9,26 +9,12 @@
 
 import * as fs from 'node:fs';
 import * as path from 'node:path';
-import * as url from 'node:url';
 
 import {Benchmarker, applySettings, assert, loadSettingsFile, printErr, read} from './utility.mjs';
 
-function find(filename) {
-  assert(filename);
-  const dirname = url.fileURLToPath(new URL('.', import.meta.url));
-  const prefixes = [dirname, process.cwd()];
-  for (let i = 0; i < prefixes.length; ++i) {
-    const combined = path.join(prefixes[i], filename);
-    if (fs.existsSync(combined)) {
-      return combined;
-    }
-  }
-  return filename;
-}
-
 // Load default settings
-loadSettingsFile(find('settings.js'));
-loadSettingsFile(find('settings_internal.js'));
+loadSettingsFile('settings.js');
+loadSettingsFile('settings_internal.js');
```

Comment:
loadSettingsFile already calls `find` internally so there was no need for this call.